torchmultimodal/models/albef/model.py

```diff
@@ -167,12 +167,12 @@ def __init__(
         remove_grad(self.text_proj_m)
 
         self.queue_size = queue_size
-        self.temp = temp
+        self.temp = nn.Parameter(torch.ones([]) * temp)
```

Comment:
Is the purpose to create a scalar tensor? 
Why not just do `torch.tensor(temp)`?
```
>>> nn.Parameter(torch.tensor(0.7))
Parameter containing:
tensor(0.7000, requires_grad=True)
>>> nn.Parameter(torch.ones([]) * 0.7)
Parameter containing:
tensor(0.7000, requires_grad=True)
```

Reply:
Yeah I often see scalar parameters constructed this way (see e.g. [CLIP](https://github.com/mlfoundations/open_clip/blob/1eb2fcfbc1e1c80b94fd63130f4811f09af20c8b/src/open_clip/model.py#L395)). I think it is more for legacy reasons than anything, see the discussion [here](https://discuss.pytorch.org/t/best-way-to-define-a-scalar-using-nn-parameter-in-pytorch/97209). Will change it for clarity